client/client_test.go

```diff
@@ -193,81 +191,116 @@ func TestHttpClientUrl(t *testing.T) {
 }
 
 // Test that our loaded certificates include the system CAs, and our own.
-func TestCaLoading(t *testing.T) {
-	conf := Config{
-		ServerCert: "server.crt",
-	}
-
-	certs := loadServerTrust(&conf)
-
-	// Verify that at least one of the certificates belong to us, and one
-	// belongs to a well known certificate authority.
-	var systemOK, oursOK bool
-	subj := certs.Subjects()
-	for i := 0; i < len(subj); i++ {
-		if strings.Contains(string(subj[i]), "VeriSign, Inc.") {
-			systemOK = true
-		}
-		// "Acme Co", just a dummy certificate in this repo.
-		if strings.Contains(string(subj[i]), "Acme Co") {
```

Comment:
There is no call that can easily be used to get the default location, but we do not need that, it is enough to call `ctx.SetDefaultVerifyLocations()` now, as all is merged (see [above comment](https://github.com/mendersoftware/mender/pull/588#discussion_r472117054))

Reply:
Unless you have some strong objections here, @kacf I'm going to resolve this. 

So in short: It is annoyingly difficult to do now, but I'm sure we can do it. Depends on how hard of a requirement it is

Reply:
It sounds like the effort / payoff ratio is too high here. We will catch errors here eventually, because Hosted Mender will not work if this doesn't work, and testing Hosted Mender is part of our manual tests. It just won't be tested automatically. So I'm ok with resolving this.